Tests/test_imagedraw.py

```diff
@@ -400,6 +400,28 @@ def test_floodfill_border(self):
         self.assert_image_equal(
             im, Image.open("Tests/images/imagedraw_floodfill2.png"))
 
+
+    @unittest.skipIf(hasattr(sys, 'pypy_version_info'),
```

Comment:
Sorry, I just used that decorator because I saw that the other floodfill test required it.  Will remove.